### DISCUSSION TITLE
Json schema validation

### DISCUSSION THREAD

**Discussion by anyulled** [jreleaser/jreleaser#243] [Ideas]
Hi again! I've seen in Intellij IDEA that you can validate either the YAML or the JSON configuration file for JReleaser. I'd work on a JSON schema validator based on the documentation if you agree. That would ease the library configuration when working with an IDE.

**Comment by aalmiray** [comment]
That'd be super useful! I added `yamllint` for validating the structure of the YAML file but did not find a suitable option for schema validation. Left out JSON validation for the same reason. Thanks!
